web/Dockerfile.okteto

```diff
@@ -0,0 +1,15 @@
+FROM node:10
+EXPOSE 3000 9229
+
+ADD ./package.json /src/package.json
+ADD ./yarn.lock /src/yarn.lock
+WORKDIR /src
+RUN yarn install --silent --frozen-lockfile
+
+ADD . /src/
+WORKDIR /src
```

Comment:
There is another WORKDIR above.  This one can be remove.  Also, you could put WORKDIR at the top and simplify your COPY/ADD by remove the "/src"

Reply:
This is just a copy of the current Skaffold dockerfile. Thoughts on keeping iterating after merge?